### PR TITLE
De-gridify

### PR DESCRIPTION
**Pull request by matyasselmeci**
SOFTWARE-5434.  Got rid of "grid" except in code blocks and host names.  Changing VO -> collaboration will be done in a separate PR.

### CODE REVIEW

File: docs/worker-node/install-wn-oasis.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ Installing the Worker Node Client via OASIS
 ===========================================
 
 The **OSG Worker Node Client** is a collection of software components that is expected to be added to every worker node
-that can run OSG jobs. It provides a common environment and a minimal set of common tools that all OSG jobs can expect
+that can run OSPool jobs. It provides a common environment and a minimal set of common tools that all OSPool jobs can expect
```
Comment (brianhlin) [inline docs/worker-node/install-wn-oasis.md:8]: I think OSG jobs is actually ok here. It's not just OSPool, the WN client is a combination of requirements from all the collabs

File: docs/worker-node/install-wn-oasis.md [hunk 2]
```diff
@@ -26,7 +26,7 @@ As with all OSG software installations, there are some one-time (per host) steps
 -   Ensure the host has [a supported operating system](../release/supported_platforms.md)
 -   On every worker node, [install and configure CVMFS](install-cvmfs.md)
 
-Once configured to use OASIS, grid jobs will download the worker-node software on demand (into the local disk cache).
+Once configured to use OASIS, OSPool jobs will download the worker-node software on demand (into the local disk cache).
```
Comment (brianhlin) [inline docs/worker-node/install-wn-oasis.md:29]:
```suggestion
Once configured to use OASIS, OSPool jobs will download the worker-node software on demand (into the local disk cache).
```

Same thing as above, not limited to OSPool

**Review by brianhlin — APPROVED** [review]
Some minor comments/suggestions, pre-approving

File: docs/common/registration.md [hunk 3]
```diff
@@ -39,7 +39,8 @@ To register your contact information with the OSG Consortium, follow the instruc
 Registering Resources
 ---------------------
 
-An OSG resource is a host that provides grid services, e.g. Compute Entrypoints, storage endpoints, or perfSonar hosts.
+An OSG resource is a host that provides services to OSG campuses and collaborations;
+some examples are Compute Entrypoints, storage endpoints, or perfSonar hosts.
```
Comment (osg-cat) [inline docs/common/registration.md:43]: It should be “perfSONAR” throughout — that’s how they write it.

File: docs/common/registration.md [hunk 4]
```diff
@@ -52,7 +53,7 @@ OSG resources are stored under a hierarchy of facilities, sites, and resource gr
     i.e. all resources associated with a specific computing cluster.
     Multi-resource downtimes are easiest to declare across a resource group.
     Production and testing resources must be placed into separate resource groups.
--   **Resource**: A host that provides grid services, e.g. Compute Entrypoints, storage endpoints, or perfSonar hosts.
+-   **Resource**: A host that provides services, e.g. Compute Entrypoints, storage endpoints, or perfSonar hosts.
```
[the remark this answers is not in the document]
Comment (osg-cat) [reply docs/common/registration.md:56]: I like the original “services” — it is generic.

File: docs/compute-element/htcondor-ce-overview.md [hunk 5]
```diff
@@ -47,25 +47,25 @@ Benefits of running the HTCondor-CE:
...
 ### Hosted CE over SSH
 
-The Hosted CE is intended for small sites or as an introduction to the OSG.
-The OSG configures and maintains an HTCondor-CE on behalf of the site.
+The Hosted CE is intended for small sites or as an introduction to providing capacity to collaborations.
+OSG staff configure and maintain an HTCondor-CE on behalf of the site.
 The Hosted CE is a special configuration of HTCondor-CE that can submit jobs to a remote cluster over SSH.
-It provides a simple starting point for opportunistic resource owners that want to start contributing to the OSG with
-minimal effort: an organization will be able to accept OSG jobs by allowing SSH access to a login node in their cluster.
+It provides a simple starting point for opportunistic resource owners that want to start contributing to collaborations with
```
Comment (osg-cat) [inline docs/compute-element/htcondor-ce-overview.md:65]: Why not just the approved phrase of “… contributing capacity …”?

File: docs/compute-element/htcondor-ce-overview.md [hunk 6]
```diff
@@ -131,7 +131,7 @@ installation, there are two main ways to customize your CE (if you decide any cu
 How Security Works
 ------------------
 
-In the OSG, communication is secured between various parties using a combination of PKI infrastructure involving
+Communication is secured between various parties using a combination of PKI infrastructure involving
```
Comment (osg-cat) [inline docs/compute-element/htcondor-ce-overview.md:134]: I feel like starting with “Communication” is too broad. In the spirit of the original, maybe “Among OSG services, communication …”?

File: docs/compute-element/install-htcondor-ce.md [hunk 7]
```diff
@@ -220,8 +220,8 @@ For information on how to troubleshoot your HTCondor-CE, please refer to the ups
 Registering the CE
 ------------------
 
-To contribute to the the OSG Production Grid, your CE must be
-[registered with the OSG](https://github.com/opensciencegrid/topology#topology).
+To contribute capacity to collaborations, your CE must be
```
Comment (osg-cat) [inline docs/compute-element/install-htcondor-ce.md:223]: As above, maybe just drop “to collaborations”?

File: docs/compute-element/htcondor-ce-overview.md (hunk 6 above)
Comment (matyasselmeci) [reply docs/compute-element/htcondor-ce-overview.md:134]: So "OSG services" is allowed?

File: docs/worker-node/install-wn-oasis.md (hunk 1 above)
Comment (matyasselmeci) [reply docs/worker-node/install-wn-oasis.md:8]: OK, if "OSG jobs" is allowed then I'll keep it.

File: docs/data/update-oasis.md [hunk 8]
```diff
@@ -7,7 +7,7 @@ Updating Software in OASIS
 OASIS is the OSG Application Software Installation Service that can be used to publish and update software on OSG Worker
 Nodes under `/cvmfs/oasis.opensciencegrid.org`.
 It is implemented using CernVM FileSystem (CVMFS) technology and is the recommended method to install software on the
-Open Science Grid.
+OSG Fabric of Services.
```
[the remark this answers is not in the document]
Comment (matyasselmeci) [reply docs/data/update-oasis.md]: Fixed

File: docs/worker-node/install-wn-oasis.md (hunk 2 above)
Comment (matyasselmeci) [reply docs/worker-node/install-wn-oasis.md]: Fixed

File: docs/compute-element/htcondor-ce-overview.md (hunk 5 above)
Comment (matyasselmeci) [reply docs/compute-element/htcondor-ce-overview.md]: Fixed

File: docs/compute-element/install-htcondor-ce.md (hunk 7 above)
Comment (matyasselmeci) [reply docs/compute-element/install-htcondor-ce.md]: Fixed

File: docs/common/registration.md (hunk 3 above)
Comment (matyasselmeci) [reply docs/common/registration.md]: Fixed

File: docs/compute-element/htcondor-ce-overview.md (hunk 6 above)
Comment (matyasselmeci) [reply docs/compute-element/htcondor-ce-overview.md]: Assuming so, then fixed using your phrasing.

**Pull request merged**